02-peer-protocol.md

```diff
@@ -2773,18 +3435,34 @@ A node:
 
 A receiving node:
   - if `next_funding_txid` is set:
```

Comment:
Shouldn't this be
```
  - if `next_funding` is set:
```
Since including the TLV is optional but including `next_funding_txid` inside the TLV is not optional?